src/main/java/de/tum/www1/orion/build/instructor/OrionInstructorBuildUtil.kt

```diff
@@ -40,7 +43,9 @@ enum class RepositoryCheckoutPath : CustomizableCheckoutPath {
         override fun forProgrammingLanguage(language: ProgrammingLanguage): String {
             return when (language) {
                 ProgrammingLanguage.JAVA, ProgrammingLanguage.PYTHON -> ""
-                ProgrammingLanguage.C -> return "tests"
+                ProgrammingLanguage.C -> "tests"
+                // placeholder, runTestsLocally prevents any other language from reaching this line anyways
+                else -> ""
```

Comment:
If this is never reached, we should throw an exception here, so we know if when messed it up.

Reply:
resolved :+1: